userenvs/fedora31.json

```diff
@@ -0,0 +1,41 @@
+{
+    "workshop": {
+	"schema": {
```

Comment:
I wonder if we need "schema" here or just have "version" one level up.  See this example: https://coreos.com/ignition/docs/latest/examples.html

Reply:
This may or may not have been a case of premature optimization on my part.  I was trying to be prepared for the potential future addition of other properties by have this in a clearly segmented section.

Reply:
To be clear, I'm not completely opposed to changing it if we think that is the right thing to do.  At this point that doesn't involve that much work (there are only a few workshop.json files that need updating) but down the road that may be the case if we decide we need to add it back for some reason (since the number of workshop.json files will presumably have multiplied significantly).

Reply:
The idea I had for the future was the addition of a additional properties inside `workshop` that could be used to alter workshop.pl behavior (such as always doing distro updates -- or never doing them).  Logically that seems different to me that schema related properties hence the segmentation.  Of course that is all based on the idea of behavior that doesn't exist yet.

Reply:
Something else that just occurred to me is that under `schema` we may need to eventually add a property that has a URL where the schema can reliably be sourced from:

```
{
    "workshop": {
        "schema": {
            "version": "2020-03-02",
            "url": "https://raw.githubusercontent.com/perftool-incubator/workshop/master/schema.json"
        }
    }
...
```

This would allow the validator to be used against a JSON when the workshop repo is not readily available.  This type of behavior would be very similar to what I have seen in XML type validations (such as in libvirt in the past).

Reply:
OK, we can keep schema